org.bridgedb/src/test/java/org/bridgedb/DataSourcePatternsTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ *BridgeDb,
+ *An abstraction layer for identifier mapping services, both local and online.
+ *Copyright (c) 2012 Egon Willighagen <egonw@users.sf.net>
+ *Copyright (c) 2012 OpenPhacts
```

Comment:
I don't think there is any code left from the previous session. Please update this line to just list your name. If you do prefer not to give your email address, you can use `<youphendriks@github.com>`, but likely your email address is in the git commit message anyway.